nix/cardano-services/deployments/default.nix

```diff
@@ -225,55 +225,23 @@ in
           };
         };
 
-        projectors = {
-          stake-pool.enabled = true;
-        };
-
-        values = {
-          blockfrost-worker.enabled = false;
-          pg-boss-worker.enabled = true;
-
-          backend.hostnames = ["backend.${final.namespace}.eks.${baseUrl}" "${final.namespace}.${baseUrl}"];
-          backend.allowedOrigins = lib.concatStringsSep "," allowedOriginsDev;
-          backend.routes = let
-            inherit (oci.meta) versions;
-          in
-            lib.concatLists [
-              (map (v: "/v${v}/health") versions.root)
-              (map (v: "/v${v}/live") versions.root)
-              (map (v: "/v${v}/meta") versions.root)
-              (map (v: "/v${v}/ready") versions.root)
-              (map (v: "/v${v}/asset") versions.assetInfo)
-              (map (v: "/v${v}/chain-history") versions.chainHistory)
-              (map (v: "/v${v}/network-info") versions.networkInfo)
-              (map (v: "/v${v}/rewards") versions.rewards)
-              (map (v: "/v${v}/tx-submit") versions.txSubmit)
-              (map (v: "/v${v}/utxo") versions.utxo)
-              (map (v: "/v${v}/handle") versions.handle)
-              (map (v: "/v${v}/provider-server") versions.stakePool)
-              (map (v: "/v${v}/stake-pool-provider-server") versions.stakePool)
-            ];
+        "dev-sanchonet@us-east-1@v1" = final: {
```

Comment:
@bernokl , can you take a look at these changes from the merge?
I am unsure wether they will mess any of the deployment work.